examples/evaluator-tests/test_pddl_evaluator.py

```diff
@@ -12,7 +11,7 @@
 parser.add_argument("problem", type=str)
 args = parser.parse_args()
 
-script_dir = os.path.dirname(tools.get_script_path())
+script_dir = tools.get_script_path().parent
```

Comment:
Not really related to this issue but we probably want this directory at least as often as the path to the script itself. We could add the following function to `tools.py`:
```Python
def get_script_dir():
    return get_script_path().parent
```